script.js

```diff
@@ -0,0 +1,12 @@
+const menu = document.querySelector('.menu-bars');
+const mobileNav = document.querySelector('.mobile-nav');
+const cross = document.querySelector('.cross');
+
+if (menu) {
+  menu.addEventListener('click', () => {
+    mobileNav.classList.toggle('hide');
+    cross.classList.toggle('hide');
+  });
+} else {
+  console.log('Something went wrong');
+}
```

Comment:
As the requirement your hamburger icon should display the mobile menu. You have only added the functionality for closing the menu button. Pls make sure to add the required code to make the button work with it is clicked. Kindly fix this issue.